packages/gasket-plugin-happyfeet/lib/actions.js

```diff
@@ -0,0 +1,10 @@
+const happyFeet = require('happy-feet');
+
+/** @type {import('@gasket/core').HookHandler<'actions'>} */
+module.exports = function actions(gasket) {
+  return {
+    getHappyFeet(happyConfig) {
+      return happyFeet(gasket.config.happyFeet || happyConfig || {});
```

Comment:
If you end up using the same instance for the whole app you should set a single instance to reference in the action file and then return that every time you call the action.

Something like below

```
let  happy = null

module.exports = function actions(gasket) {
 if(!happy) {
happy = happyFeet(gasket.config.happyFeet || {})
}
return happy
```

Reply:
@jpina1-godaddy I had this originally, but changed it per t[his thread](https://godaddy.slack.com/archives/C034EH5GG4R/p1719248113151249?thread_ts=1719009070.161629&cid=C034EH5GG4R)

Reply:
@mmason2-godaddy Wouldn't this code create a new instance of happy-feet every time you call it?  I'm not too familiar with happy feet but I think you would want to create the one instance and then reference it later instead of creating a new one each time. Kind of like how it was before when it was on the gasket object but instead of it living on the gasket object it would reference the top level variable in this file.

Reply:
I'm not super familiar with this either. I'm good with that implementation 👍 . Sorry for the back and forth!

Reply:
No worries, thanks for taking the time to send feedback.